README.md

```diff
@@ -171,7 +171,7 @@ To use a different version of Java, please use a docker tag to run your Minecraf
 | openj9-11      | Uses Eclipse OpenJ9 JVM for Java 11         | Alpine Linux |
 | openj9-nightly | Uses Eclipse OpenJ9 JVM testing builds      | Alpine Linux |
 | multiarch      | Uses Java version 8 latest update           | Debian Linux |
-| multiarch-latest | Uses Java version 15 latest update        | Debian Linux |
+| multiarch-latest | Uses Java version 15 latest update        | Ubuntu Linux |
```

Comment:
FYI, it's still based on Debian:

```
> docker run -it --rm --entrypoint bash itzg/minecraft-server:multiarch-latest -c 'cat /etc/debian_version'
bullseye/sid
```

but in any case, this change needs to be reverted since I overhauled this table.